lesson09/src/cmd/migrate/migrate.go

```diff
@@ -0,0 +1,39 @@
+package migrate
+
+import (
+	"fmt"
+
+	_ "github.com/lib/pq"
+
+	"github.com/lunarnuts/go-course/tree/lesson09/src/db"
+	_ "github.com/lunarnuts/go-course/tree/lesson09/src/db/migrations"
```

Comment:
This won't compile as the directory does not contain any go files.
Still, even when removing this line, the migration is not performed

Reply:
I dont understand why, but git add . did not add all files.